packages/webapp/src/components/Typography/index.tsx

```diff
@@ -205,3 +207,53 @@ export const Text = ({ children = 'Text', className = '', style, ...props }: Typ
     </p>
   );
 };
+
+export const ExternalLink = ({ children = 'Link', className = '', style, link, ...props }) => {
+  return (
+    <>
+      <div
+        className={clsx(styles.text, className)}
+        style={{ width: 'fit-content', display: 'inline-block', ...style }} // I'm not sure about style...
```

Comment:
@MwayaB 

I'm not sure about styles either lol (probably should remove this comment). 

If anything base element styles like div's and p's etc can be put in the styles css file.

Reply:
Haha, that was in my code I shared with Mwaya.
Mwaya, please remove it lol